src/messageHandlers/getStorageData.ts

```diff
@@ -0,0 +1,13 @@
+import Background from '@/background/Background'
+
+const getStorageData = async (background: Background) => {
+  console.log('getStorageData background', background)
```

Comment:
I get this is a dummy example, but to show case the usage, you can invoke the background.storageService and return the result